test/deploy/kubernetes/deploy.sh

```diff
@@ -22,6 +22,31 @@ function show_usage {
     exit 1
 }
 
+function wait_for_pod_to_be_ready(){
```

Comment:
**Improvement:**
We should make function generic as in:
check_pod_status(message_header, filter_args, status_list):
message_header: will display message like "Check for pod `filter_args` with status `status_list`"
filter_args: Will be used to filter pods (Example: `data-node.-.*`)
status_list: List of status polling

Reply:
Will take it up later.